pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -933,6 +933,10 @@ const (
 	// should wait with reconciliation of the operating system config (to prevent too many node-agents from restarting
 	// kubelet or other critical units at the same time).
 	AnnotationNodeAgentReconciliationDelay = "node-agent.gardener.cloud/reconciliation-delay"
+	// NodeAgentsGroup is the identity group for node-agents when authenticating to the API server.
```

Comment:
```suggestion
	// NodeAgentsGroup is the identity group for gardener-node-agents when authenticating to the API server.
```

Consistency nit, since in the comment below it is `gardener-node-agent`. Or you could remove the `gardener-` prefix from both.